ccmbioinfo/lfs
# CARDS-1821: Deprecate the cards4care project

The backend code has been moved form cards4care to heracles.

**Testing**
* Ensure date obfuscation works in heracles
* Ensure export to S3 bucket works for heracles

Testing instructions for the s3 endpoint are available in https://github.com/data-team-uhn/cards/pull/491.
* Note: The command  to run `minio` has changed since the linked PR, it is now:
`docker run --rm -p 9000:9000 -p 9001:9001 minio/minio server /data --console-address ":9001`
